api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/PersonAttributeDataConverter.java

```diff
@@ -24,8 +26,26 @@ public Object convert(Object obj) {
         if (obj == null) {
             return "";
         }
-
-        return ((PersonAttribute)obj).getValue();
+        String personAttribute = ((PersonAttribute)obj).getValue();
+        if(personAttribute!=null)
+        {
+            if(personAttribute.equals("165317"))
```

Comment:
@solemabrothers No need for else if, you can just have multiple stand along if statements

Reply:
Done